src/main/resources/.data/carpet-redcraft-addons/scripts/specialnametags.sc

```diff
@@ -17,6 +17,9 @@ __on_player_interacts_with_entity(player, entity, hand) -> (
             parti = slice(split('&', custom_name),1);
             modify(entity, 'custom_name', format(parti))
         );
+        if(entity ~ 'type' == 'armor_stand',
+            modify(entity, 'nbt_merge', '{CustomNameVisible:true}')
+        );
```

Comment:
È sufficiente aggiungere il quarto parametro alla modify(entity) antecedente

Reply:
Ma poi non crea problemi con altre entità??

Reply:
Mh... Non credo... In ogni modo conviene modificare con modify invece che con NBT merge: è un'operazione molto pesante